src/coresyf_speckle_filter/examples.sh

```diff
@@ -0,0 +1,4 @@
+#ASA_WSM_1PXPDE20021117_104431_000000672011_00180_03741_0009.N1.gz="http://example.org"
```

Comment:
Is this working with this line as comment?

Reply:
I uncommented the line and changed it to match the name of the input.